data/remnant missions.txt

```diff
@@ -1252,3 +1252,232 @@ mission "Remnant: Expanded Horizons Quarg 3"
+	destination
+		government "Remnant"
+	on offer
+		require "Jump Drive (Broken)"
+		conversation
+			`As you arrive in the spaceport concourse the Director looks up and waves you over.`
+			`	"Good timing, Captain <first>," he sings, "we have a team of researchers on <planet> who have requested a broken jump drive. Could you deliver one to them?"`
+			choice
+				`	(Maybe later.)`
+					goto refuse
+				`	(Yes.)`
+			`	"Excellent. Your finds have spurred discussion and excitement in various quarters. We have had teams researching jump drives for about at least a century now without a lot of success. I'm told they are rather intricate, but your finds are opening new avenues of investigation. That being said, I'll notify the team on <planet> to expect you."`
+				accept
+			label refuse
+			`	He looks disappointed, and taps something on his screen. "OK, I'll file that request away for now. Come back if you change your mind."`
+				defer
+	on complete
+		conversation
+			`The team waiting for you on <planet> is definitely eager to begin. You barely have the hatch open before they are chivying their camel-pulled wagon up the ramp. They load the broken jump drive surprisingly fast despite their delicate movements.`
+			`	"This is an exciting day, Captain <first>! Countless theories may be proven or sent back for more work based on the testing we can do with this!"`
+			choice
+				`	"Any theories of note?"`
+					goto studying
+				`	"Great! I will find more."`
+			label studying
+			`	He looks thoughtful for a moment, then warbles "As you may know, standard hyperdrives work by generating a fusion reaction and folding it into the fabric of space/time, and these folds naturally fall along the known hyperlanes. I suppose one could view those lanes as the 'crease lines' of the universe. My favorite theory is that jump drives have some additional machinery to guide the fold. That way they can dictate where the fold occurs instead of just aligning with the nearest crease. Given their increase fuel draw, it makes sense that it would take more fuel to generate a fresh fold than to re-use an existing crease."`
+			`	"Oh, before I forget.." he hands you <payment>. "Off to start setting up the tests" he trundles off after the wagon disappearing down a distant corridor.`
+		outfit "Jump Drive (Broken)" -1
+		"remnant: broken jump drive count" ++
+		payment 500000
+		"reputation: Remnant" ++
+
+
+
+mission "Remnant: Broken Jump Drive 4"
+	name "Broken Jump Drive Delivery"
+	description "You have been asked to deliver a broken jump drive to the researchers on <planet>."
+	to offer
+		has "Remnant: Broken Jump Drive 3: done"
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+	on offer
+		require "Jump Drive (Broken)"
+		conversation
+			`You head over to the Director's desk to see who is in need of the next broken jump drive. The director their smiles at you, looks over several messages on her screen, then clears her throat.`
```

Comment:
Previously the Director has been referred to with male pronouns. This may be different across planets, but there's no way to know which planets the missions start/end on.

Reply:
This is intentional. What the Remnant term as "Director" is more along the lines of "ground traffic controller" at an airport. Or perhaps "Information desk attendant." There's many, many more than just one at every spaceport, so I figure there's about a 50/50 chance the player talks to a male or female one at any particular point in time. The Remnant are aggressively egalitarian in that regards, and "Director" for them is probably more along the lines of "supervisor" in terms of rank/prestige, or perhaps mid to low level manager. No where near the level of the Prefects.

Reply:
`The director their smiles at you` --> there